addons/purchase/views/purchase_views.xml

```diff
@@ -243,10 +244,10 @@
                                            optional="hide"
                                            groups="analytic.group_analytic_accounting"
                                            options="{'product_field': 'product_id', 'business_domain': 'purchase_order', 'amount_field': 'price_subtotal'}"/>
-                                    <field name="product_qty"/>
+                                    <field name="product_qty" readonly="is_downpayment"/>
```

Comment:
You might need to add in extra `is_downpayment` logic for `date_planned`and `discount`. Should be double checked, I'm getting weird "Invalid Fields" warning if I try to set these values in my localhost + save